unified_targets/configs/TALONF7V2.config

```diff
@@ -0,0 +1,111 @@
+# Betaflight / STM32F7X2 (S7X2) 4.0.5 Aug  1 2019 / 12:29:33 (839661b20) MSP API: 1.41
+board_name TALONF7V2
+manufacturer_id TALN
```

Comment:
This is not an official manufacturer id. Please follow the instructions at https://github.com/betaflight/betaflight/blob/master/unified_targets/docs/CreatingAUnifiedTarget.md

Reply:
changed in #8649 
thanks @mikeller